Automattic/jetpack
# Block editor: add .min suffix to all resources loaded in editor

Fixes #20745

#### Changes proposed in this Pull Request:

This is not perfect though:

- It adds the suffix to asset php files as well. It's not needed there.
- It adds the suffix to CSS files too. The CSS, while automatically generated, is not actually minified.

#### Jetpack product discussion

* p93R5q-14E-p2#comment-1926

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Posts > Add New
* The block editor should still have Jetpack blocks available
* Add a block that includes frontend scripts and styles, such as the Slideshow block.
* It should look good on the frontend.
